### PR TITLE
Allow ordering as a guest, even after a you made an account

### PR DESCRIPTION
**Pull request by Hlavtox**
Everything is here - https://github.com/PrestaShop/PrestaShop/pull/25353

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]
LGTM but we need to wait for the core PR

**Review by matthieu-rolland — CHANGES_REQUESTED** [review]
I put a `request change` to be sure the PR is not merged until it can be merged, feel free to cancel it when it's time

**Review by MeKeyCool — APPROVED** [review]

**Pull request closed without merging**
